### PR TITLE
Add info box to Remote Path Mappings Settings

### PR DESCRIPTION
**Pull request by stevietv**
#### Database Migration
NO

#### Description
Add info box to Remote Path Mappings that they are rarely required

#### Issues Fixed or Closed by this PR

*

### CODE REVIEW

File: src/NzbDrone.Core/Localization/Core/en.json [hunk 1]
```diff
@@ -929,6 +929,7 @@
   "RemotePathMappingRemotePathHelpText": "Root path to the directory that the Download Client accesses",
   "RemotePathMappingWrongOSPathHealthCheckMessage": "Remote download client {0} places downloads in {1} but this is not a valid {2} path. Review your remote path mappings and download client settings.",
   "RemotePathMappings": "Remote Path Mappings",
+  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required, it is better to match your download path client to your local paths.",
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Localization/Core/en.json:932]:
```suggestion
  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required. If you have {app} in Docker and the Download Client in non-Docker (or vice versa) or have the programs on different servers then you may need a remote path map.",
```
updte translate implementation accordingly of course

source - https://wiki.servarr.com/sonarr/troubleshooting#remote-path-mapping

Comment (stevietv) [reply src/NzbDrone.Core/Localization/Core/en.json:932]: maybe better to link to wiki instead of a long info box? open to others thoughts also!

Comment (markus101) [inline src/NzbDrone.Core/Localization/Core/en.json:932]:
```suggestion
  "RemotePathMappingsInfo": "Remote Path Mappings are very rarely required, if {app} and your download client are on the same system it is better to match your download client path. For more information see {wikiLink}",
```

I don't think we need to spell out every possible combination in the info text, if people don't want to follow a link to the wiki they're probably not reading it in the UI either.

Comment (stevietv) [reply src/NzbDrone.Core/Localization/Core/en.json:932]: incorporated this with a slight tweak

Comment (bakerboy448) [reply src/NzbDrone.Core/Localization/Core/en.json]: Troubleshooting section has more details and links back to settings.

File: frontend/src/Settings/DownloadClients/RemotePathMappings/RemotePathMappings.js [hunk 2]
```diff
@@ -50,6 +52,11 @@ class RemotePathMappings extends Component {
           errorMessage={translate('RemotePathMappingsLoadError')}
           {...otherProps}
         >
+
+          <Alert kind={kinds.INFO}>
+            <InlineMarkdown data={translate('RemotePathMappingsInfo', { app: 'Sonarr', wikiLink: 'https://wiki.servarr.com/sonarr/settings#remote-path-mappings' })} />
```
Comment (bakerboy448) [inline frontend/src/Settings/DownloadClients/RemotePathMappings/RemotePathMappings.js:57]:
```suggestion
            <InlineMarkdown data={translate('RemotePathMappingsInfo', { app: 'Sonarr', wikiLink: 'https://wiki.servarr.com/sonarr/settings#remote-path-mappings' })} />
```

File: src/NzbDrone.Core/Localization/Core/en.json (hunk 1 above)
Comment (stevietv) [reply src/NzbDrone.Core/Localization/Core/en.json]: > Troubleshooting section has more details and links back to settings.

not sure we should be directly sending someone to troubleshoot for general info about what Remote Path Mappings are

**Review by markus101 — APPROVED** [review]

**Pull request merged**